domains/rameez.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Personal website of Rameez",
+  "repo": "https://github.com/rameezrami/rameezrami.github.io",
+  "owner": {
+    "username": "rameezrami",
+    "email": "rameespu@gmail.com"
+  },
+  "record": {
+    "URL": "https://rameez.is-a.dev"
```

As I mentioned in the old PR, `rameez.is-a.dev` cant be redirected to `https://rameez.is-a.dev`.

For URL redirection
```suggestion
    "URL": "https://rameezrami.github.io"
```

For subdomain
```suggestion
    "CNAME": "rameezrami.github.io"
```